editor.planx.uk/src/pages/FlowEditor/components/Team/components/EditorUpsertModal.tsx

```diff
@@ -39,12 +47,23 @@ export const AddNewEditorModal = ({
     values: AddNewEditorFormValues,
     { resetForm }: FormikHelpers<AddNewEditorFormValues>,
   ) => {
+    switch (actionType) {
+      case "add":
+        handleNewSubmit();
```

Comment:
nit: I'd name this even more explictly, e.g. `handleSubmitToAddNewUser` or `handleAddNewUser`